app/controllers/sessions_controller.rb

```diff
@@ -1,2 +1,28 @@
 class SessionsController < ApplicationController
+  before_action :set_seller, only: [:create]
+
+  def new
```

Comment:
This should be using the single-line syntax that the other methods with no body are using